code/modules/mob/living/carbon/human/human_examine.dm

```diff
@@ -57,15 +57,11 @@
 	else if(displayed_species == "Slime People") //snowflakey because Slime People are defined as a plural
 		msg += ", a<b><font color='[examine_color]'> slime person</font></b>!"
 	else
-		// do all this extra stuff because byond's text macros get confused by whatever comes between the species name and the article,
-		// so we can't just do \a
-		var/article_override = dna?.species.article_override
-		var/article = article_override
-		if(!article_override)
-			article = starts_with_vowel(displayed_species) ? "an" : "a"
-
-		msg += ", [article]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b>!"
+		var/article = "a"
+		if(height == "average height" || physique == "average build" || physique == "overweight" )
+			article = "an"
 
+		msg += ", [article] [height]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b> with [article] [physique] physique!" //Look I know it uses physique twice, lets not think about it too much ok?
```

Comment:
Since the articles are directly referring to the height and physique, we can just use text macros.
```suggestion
		msg += ", \a [height]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b> with \a [physique] physique!" //Look I know it uses physique twice, lets not think about it too much ok?
```